olp-cpp-sdk-authentication/src/AuthorizeRequest.cpp

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright (C) 2020 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#include "olp/authentication/AuthorizeRequest.h"
+
+#include <sstream>
+
+namespace olp {
+namespace authentication {
+
+std::string AuthorizeRequest::CreateKey() const {
```

Comment:
You should be able to inline this inside the header and save the cpp file.

Reply:
@mykhailo-kuchma doesn't like additional includes as sstream in header file(https://github.com/heremaps/here-olp-sdk-cpp/pull/792#discussion_r411947706). And suggest to remove this function at all. What do you think, should we remove it?

Reply:
We used the inline approach in dataservice-read.
But it doesn't matter very much. I guess it is fine like this also.